src/palace/manager/api/admin/model/quicksight.py

```diff
@@ -7,13 +7,16 @@
 
 
 class QuicksightGenerateUrlRequest(CustomBaseModel):
-    library_uuids: list[UUID] = Field(
-        description="The list of libraries to include in the dataset, an empty list is equivalent to all the libraries the user is allowed to access."
+    library_uuids: list[UUID] | None = Field(
```

Comment:
Is this change needed? It looks like the new validator always makes sure that its always not None. Seems like we don't need to change the type hint here.